kernel/object-store/src/main/java/io/mosip/commons/khazana/impl/S3Adapter.java

```diff
@@ -442,27 +430,35 @@ private AmazonS3 getConnection(String bucketName) {
 		 String finalObjectName=null;
      	if(useAccountAsBucketname) {
      		 bucketName=account;
-     		 finalObjectName = ObjectStoreUtil.getName(container,null,TAGS_FILENAME);
+     		 finalObjectName = ObjectStoreUtil.getName(container,null,TAGS_FILENAME)+SEPARATOR;
      	}else {
      		 bucketName=container;
-     		 finalObjectName = TAGS_FILENAME;
+     		 finalObjectName = TAGS_FILENAME+SEPARATOR;
      	}
 		AmazonS3 connection = getConnection(bucketName);
-		if(connection.doesObjectExist(bucketName, finalObjectName)) {
 		
-		GetObjectTaggingRequest getObjectTaggingRequest=new GetObjectTaggingRequest(bucketName,finalObjectName);
-		
-			GetObjectTaggingResult getObjectTaggingResult=connection.getObjectTagging(getObjectTaggingRequest);
-		
-		if(getObjectTaggingResult!=null) {
-			List<Tag> tagSet = getObjectTaggingResult.getTagSet();
-			if (tagSet != null) {
-				for(Tag tag:tagSet) {
-					objectTags.put(tag.getKey(), tag.getValue());
-				}
-			}
-		  }
-		}
+		List<S3ObjectSummary> os = null;

```

Comment:
why is this variable called os ?

Reply:
Fix :updated variable name to objectSummary